app/src/ui/changes/commit-message.tsx

```diff
@@ -489,6 +498,7 @@ export class CommitMessage extends React.Component<
           className="commit-button"
           onClick={this.onSubmit}
           disabled={!buttonEnabled}
+          tooltip={this.commitButtonTooltipText()}
```

Comment:
Why not inline the ternary?

Reply:
i prefer to avoid ternaries inline in react component props for readability even though it adds another method